frontend/src/app/components/navbar/navbar.component.html

```diff
@@ -28,15 +35,15 @@ <h1>chiever</h1>
                 </div>
 
                 <p-divider *ngIf="loggedIn" />
-            <!--A section for the register, login and logout buttons-->
-            <div style="display: flex; justify-content: space-between; gap: 1em; margin-top: 10px;">
-                <p-button [rounded]="true" [outlined]="true" label="Light" icon='pi pi-bolt' (click)="swapTheme()"
-                onLabel="Dark" offLabel="Light" />
-                <p-button [rounded]="true" [outlined]="true" *ngIf="loggedIn" label="Sign-Out" icon='pi pi-sign-out'
-                    [routerLink]="['/signout']" />
-            </div>
-        </p-overlayPanel>
-    </p-toolbar>
+                <!--A section for the register, login and logout buttons-->
+                <div style="display: flex; justify-content: space-between; gap: 1em; margin-top: 10px;">
+                    <p-button [rounded]="true" [outlined]="true" [label]="isDarkMode ? 'Dark' : 'Light'"
```

Comment:
onLabel and offLabel are not properties of p-button, so the fix I have had in main for a while is necessary